spam/01_spam_tutorial.py

```diff
@@ -214,11 +243,19 @@ def check_out(x):
 
 
 # %% [markdown]
-# To apply one or more LFs that we've written to a collection of data points, we use an `LFApplier`. Because our data points are represented with a Pandas DataFrame in this tutorial, we use the `PandasLFApplier`.
+# To apply one or more LFs that we've written to a collection of data points, we use an
+# [`LFApplier`](https://snorkel.readthedocs.io/en/redux/source/snorkel.labeling.apply.html#snorkel-labeling-apply-package).
+# Because our data points are represented with a Pandas DataFrame in this tutorial, we use the
+# [`PandasLFApplier`](https://snorkel.readthedocs.io/en/redux/source/snorkel.labeling.apply.html#snorkel.labeling.apply.pandas.PandasLFApplier).
```

Comment:
Completely random thought: It would be nice if the notebook test can also test that all links are valid (to e.g. be robust to function name changes).

Reply:
Agreed, meant to change all of these to master instead of redux. @vincentschen brought this up too. In theory, checking if `urllib.request.urlopen(req).code >= 400` should do the trick, but idk how good most sites are about response codes